interchain-token-service/src/lib.rs

```diff
@@ -1,5 +1,8 @@
 mod primitives;
-
-pub mod error;
 pub use primitives::*;
+
 pub mod abi;
```

Comment:
does this need to be public?

Reply:
Yeah, non smart contract chains like XRPL will implement ITS edge contract functionality in their gateways, so they need the `abi_encode`/`abi_decode` methods.